packages/desktop-client/src/components/schedules/ScheduleDetails.jsx

```diff
@@ -657,7 +667,7 @@ export function ScheduleDetails({ id, transaction }) {
                   htmlFor="form_posts_transaction"
                   style={{ userSelect: 'none' }}
                 >
-                  Automatically add transaction
+                  <Trans>Automatically add transaction</Trans>
```

Comment:
There's a few instances of text above that might be worth translating:
- "Repeats" on line 641
- "Upcoming dates" on line 608